### PR TITLE
add docs CI

### PR DESCRIPTION
**Pull request by sarahschwartz**
This PR includes:
 - a docs CI to prevent the docs-hub from breaking
 - fixes for the docs-hub

You can find a README for the CI [here](https://github.com/FuelLabs/github-actions/blob/master/docs-hub/README.md).

### CODE REVIEW

**Comment by ra0x3** [comment]
@sarahschwartz This PR still a WIP? We're releasing a pretty big 0.19 soon and I was wondering if this might make it into that (totally fine if you want to hold off on it) :)

**Comment by sarahschwartz** [comment]
@ra0x3 Yes this will hopefully be ready today, thanks for asking!

**Review by ra0x3 — APPROVED** [review]
LGTM. Will defer to @deekerno

**Pull request merged**
